src/main/java/com/algorand/algosdk/builder/transaction/KeyRegistrationTransactionBuilder.java

```diff
@@ -92,8 +101,11 @@ public T participationPublicKey(byte[] pk) {
      * @return This builder.
      */
     public T participationPublicKeyBase64(String pk) {
-        this.votePK = new ParticipationPublicKey(Encoder.decodeFromBase64(pk));
+        if(!pk.isEmpty()){
```

Comment:
These empty checks should not be needed. And they are actually a breaking change, since now `.participationPublicKeyBase64("abc...").participationPublicKeyBase64("")` produces a different result.